### PR TITLE
refactor: align template generation with output changes

### PR DESCRIPTION
**Pull request by roleyfoley**
## Description
Fixes/Refactors executor run tasks which rely on output from blueprint generation 

## Motivation and Context
With the changes implemented in #105 the filenaming for blueprint files has been handed over to the engine to control. As a result of this the file name outputs have changed sightly. This PR resolves this and also sets a standard way to manage the generation of the blueprint files 

## How Has This Been Tested?
Tested locally 

## Types of changes
- [X] Bug fix (non-breaking change which fixes an issue)
- [X] Refactor (non-breaking change which improves the structure or operation of the implementation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Followup Actions
- [x] None

## Checklist:
- [ ] My change requires a change to the [documentation](https://github.com/hamlet-io/docs).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] None of the above.

### CODE REVIEW

File: cli/runLambda.sh [hunk 1]
```diff
@@ -52,6 +55,9 @@ function options() {
             l)
                 INCLUDE_LOG_TAIL="true"
                 ;;
+            g)
```
Comment (ml019) [inline cli/runLambda.sh:58]: I don't see a corresponding changes to getopts?

File: cli/runSentryRelease.sh [hunk 2]
```diff
@@ -55,6 +55,9 @@ function options() {
     # Parse options
     while getopts ":hm:p:r:su:" opt; do
```
Comment (ml019) [inline cli/runSentryRelease.sh:56]: I don't see a corresponding changes to getopts?

**Review by ml019 — CHANGES_REQUESTED** [review]
Couple of minor fixes

File: cli/runLambda.sh (hunk 1 above)
Comment (roleyfoley) [reply cli/runLambda.sh:58]: Fixed

File: cli/runSentryRelease.sh (hunk 2 above)
Comment (roleyfoley) [reply cli/runSentryRelease.sh]: Fixed

**Pull request merged**